app/views/kurasus/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="container mt-3 text-center">
+<div class="container mt-3 text-center text-white">
```

Comment:
I mentioned on Friday that I would work on the styling of the notebook and /kurasu page. Let's not work on the same things, or our branch will have so many conflicts.